en/scripting/modules/index.md

```diff
@@ -1,88 +1,86 @@
+The editor modules are all under the `'cce:'` protocol (**cce** stands for "**C**ocos**C**reator**E**ditor").
 
-Except for module `cce.env`, all modules are visible only under editor environments. For example, they are not visible from previewing and after building. Instead, they are visible from scene editor.
+With the only exception being `'cce.env'`, all editor modules are only available in the editor environment. For example, the editor module cannot be accessed in the preview and build environment, on the contrary, it can be accessed in the **scene editor**.
 
-| Module name | Description                    |
-| :---------- | :----------------------------- |
-| `'cce.env'` | Accessing build-time constants |
+| Module name | Description |
+| :---------- | :-------------- |
+| `'cce.env'` | Used to access build-time constants |
 <!--
-| `'cce:gizmo'` | Gizmo                          |
+| `'cce:gizmo'` | Gizmo |
 -->
 
-### Build-time constants
+### Build time constant
 
-Editor module `'cce.env'` exposes some **constants** that are came from building environment. These constants may present execution environment, debugging level, platform identification and so on. Unlike other editor modules, `'cce.env'` is visible from non-editor environments.
+The editor module, `'cce.env'`, exposes some **constants** at build time. These constants represent the execution environment, debugging level, or platform identification. Unlike other editor modules, `'cce.env'` allows access in a non-editor environment.
 
-Since these constants are declared with `const` qualifier, it's very friendly to code optimization.
+As these constants are declared with `const`, it provides a good opportunity for code optimization.
 
 #### Execution environment
 
-| Name (all in type of `boolean`) | Description   |
-| :-------- | :--------------------------------- |
-| `BUILD`   | Is executing after building        |
-| `PREVIEW` | Is executing during previewing     |
-| `EDITOR`  | Is executing in editor environment |
+| Name (all types are `boolean`) | Description |
+| :-------- | :------------------- |
+| `BUILD` | Is it running after build |
+| `PREVIEW` | Is it running in preview |
+| `EDITOR` | Is it running in the editor |
 
-#### Debugging level
+#### Debug level
 
-| Name (all in type of `boolean`) | Description  |
-| :------ | :------- |
-| `DEBUG` | Is under debug mode. `false` if debug option is set when do building, `true` otherwise. |
-| `DEV`   | Equivalent to `DEBUG`/`EDITOR`/`PREVIEW`.  |
+| Name (all types are `boolean`) | Description |
+| :------ | :------ |
+| `DEBUG` | Whether it is in debug mode. It is `false` only when the debug option is not checked when building, and it is `true` in all other cases |
+| `DEV` | Equivalent to `DEBUG`/`EDITOR`/`PREVIEW` |
 
+#### Platform ID
 
-#### Platform identification
+The constants listed in the following table represent whether it is running on **a** or **a type** platform, and the type is all `boolean`.
```

Comment:
......running on a particular platform or class of platforms, and all types are boolean?

Reply:
sure